core/domain/rte_component_registry_test.py

```diff
@@ -78,7 +78,11 @@ def _validate_customization_arg_specs(self, customization_arg_specs):
                     ca_spec['default_value'], ca_spec['schema'],
                     apply_custom_validators=False))
 
-            if ca_spec['schema']['type'] == 'custom':
+            # Default value of SanitizedUrl obj_type may be empty. The empty
+            # string is not considered valid for this object, so we don't
+            # attempt to normalize it.
+            if ca_spec['schema']['type'] == 'custom' and (
```

Comment:
With the new approach, do you still need this check?

Reply:
`SanitizedUrl` object has a validator to check if string is empty [here](https://github.com/oppia/oppia/pull/6286/files#diff-1a892bc24b6c18dcd3edb68a2904cbc8R323). As default value if empty, check is needed. Please note that [here](https://github.com/oppia/oppia/blob/develop/core/domain/rte_component_registry_test.py#L77) and [here](https://github.com/oppia/oppia/blob/develop/core/domain/rte_component_registry_test.py#L86), we are doing same thing which is normalizing the default value. But in second case, `apply_custom_validators` is `True`.

Reply:
OK. Like I think @bansalnitish said before, you still need to check the "else" case. Everything should always be cross-checked somehow; the checks might be different in the different cases.

Reply:
Done.